### DISCUSSION TITLE
Clickable links

### DISCUSSION THREAD

**Discussion by popa13** [xournalpp/xournalpp#3675] [Ideas]
Hi,
This project is wonderful and the app is also wonderful!! I use it to teach in class and it works perfectly as I wish it would. There is somehow something I would like to know and that I didn't find after a little research in the options. Is it possible to add a clickable click to a webpage? We can insert images, but I would also like to insert clickable links to webpages. I would use this functionality to insert clickable links to Desmos graphs into my lecture notes.
Thank you,
Pierre

**Comment by netlimpopo** [comment]
Hello!
I use it for teaching as well. Clickable links are not possible right now. It is a long requestes feature. Amyone correct me, but also latexlinks are not chlickable yet. 
My workaround for textlinks ->rightclick on textfield filled with link only-> ctrl+c  
Then open browser -> crt+l (adressbar)->ctrl+v (paste). 
So In a way you can paste the whole text field and browsers recognise it as text. 
Introducing a new file format is planned which could support linking. 
Right Now no internal (within document) nor external links are possible.

**Comment by rolandlo** [comment]
There is actually a pull request https://github.com/xournalpp/xournalpp/pull/2319 for this feature, but its status is unclear to me at the moment.

**Comment by popa13** [comment]
Oh thank you for your response. I just hoped that I wouldn't have to do a copy and paste of my links. I hope they add this feature because it is really useful.

Thank you!

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
